content/advisories/2021-11-11-transaction-broadcast.zh-tw.md

```diff
@@ -0,0 +1,28 @@
+
++++
+title = "公告：對交易自動重新進行的回應"
+date = "2021-11-11"
+[ author ]
+  name = "Dogecoin Foundation"
++++
+
```

Comment:
there is a new sentence added here in the latest update, and please translate.

"Note: A previous version of this advisory used "we" to refer to a group separate to the Foundation. It has been edited to clarify."